code/modules/mob/living/carbon/human/species/necromorph/leaper.dm

```diff
@@ -317,6 +374,17 @@ It can be used to chase down a fleeing opponent, to move along long hallways qui
 		spawn(rand_between(0, 1.8 SECONDS))
 			play_species_audio(src, SOUND_ATTACK, 30, 1)
 
+/mob/living/carbon/human/proc/tailstrike_leaper_monkey(var/atom/A)
```

Comment:
tailstrike is very powerful, give this a longer cooldown

Reply:
Cooldown added.